core/agent/test/suite/taskrun.resource

```diff
@@ -39,9 +39,9 @@ Stop command monitoring
 
 Stop command monitoring and flush the database
     Flush the Redis database
-    The command is received    FLUSHALL
-    Close All Connections
-
+    The command is received    FLUSHALL 
+    Close All Connections 
+    
```

Comment:
Avoid unnecessary changes: this white space change only pollute the PR diff without adding real benefit.